### PR TITLE
Added feature overview

### PR DESCRIPTION
**Pull request by gkreyssing**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: content/en/docs/about/_index.md [hunk 1]
```diff
@@ -18,4 +18,18 @@ Eclipse _Velocitas™_ provides an end-to-end, scalable and modular development
 - Vehicle Apps skeleton and example Vehicle Apps helps to understand easily how to write a Vehicle Apps using the KUKSA VAL runtime services
 - Higher development velocity through self-contained, containerized Apps with no dependencies to E/E architecture
 
-## Concepts
\ No newline at end of file
+## Feature Overview
+
+|Feature |Python|C++ |
+|-|-|-|
+|Vehilce App Template | + | + |
```
Comment (doosuu) [inline content/en/docs/about/_index.md:25]:
```suggestion
|Vehicle App Template | + | + |

```

File: content/en/docs/about/_index.md [hunk 2]
```diff
@@ -18,4 +18,18 @@ Eclipse _Velocitas™_ provides an end-to-end, scalable and modular development
...
 - Higher development velocity through self-contained, containerized Apps with no dependencies to E/E architecture
 
-## Concepts
\ No newline at end of file
+## Feature Overview
+
+|Feature |Python|C++ |
+|-|-|-|
+|Vehilce App Template | + | + |
+|Vehilce App SDK | + | + |
```
Comment (doosuu) [inline content/en/docs/about/_index.md:26]:
```suggestion
|Vehicle App SDK | + | + |

```

File: content/en/docs/about/_index.md [hunk 3]
```diff
@@ -18,4 +18,18 @@ Eclipse _Velocitas™_ provides an end-to-end, scalable and modular development
...
+|Vehicle Model generator| + | + |
```
Comment (doosuu) [inline content/en/docs/about/_index.md:27]:
```suggestion
|Vehicle Model Generator| + | + |

```

File: content/en/docs/about/_index.md [hunk 4]
```diff
@@ -18,4 +18,18 @@ Eclipse _Velocitas™_ provides an end-to-end, scalable and modular development
...
+|DevConatiner for esay IDE setup | + | + |
```
Comment (doosuu) [inline content/en/docs/about/_index.md:28]:
```suggestion
|DevContainer for easy IDE setup | + | + |

```

**Review by doosuu — CHANGES_REQUESTED** [review]

**Review by emsameen — CHANGES_REQUESTED** [review]
I think the table is broken ?

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Review by doosuu — APPROVED** [review]
LGTM

**Pull request merged**